wdns/record_descr.c

```diff
@@ -259,6 +259,23 @@ const record_descr record_descr_array[] = {
 			rdf_end,
 		}
 	},
+
+	[WDNS_TYPE_CAA] = {
+		class_in,
+		{
+			rdf_int8,	/* flags: an unsigned int [0..255] */
+			rdf_string,	/* tag: non-zero-length sequence of
```

Comment:
Maybe `rdf_string` is not the best data type for tag?
The definition in `record_descr.h` of rdf_string says `/* length octet followed by that many octets */` is more general than in this comment.
RFC8659 says 
`Tags MAY contain ASCII characters "a" through "z", "A" through "Z",
   and the numbers 0 through 9.  Tags MUST NOT contain any other
   characters.  Matching of tags is case insensitive.`

Reply:
Section 4.1 of the rfc says it does start with a length octet. If you mean the actual content of the tag having characters outside of those ranges, my thinking was that we'd still want to output those since we're not an "enforcer" in this context.

Reply:
I meant the actual content -- maybe modify the comment to say what the RFC allows?

Reply:
Maybe modify the comment to note that we don't enforce the limitation in the RFC?